### DISCUSSION TITLE
AfterColumn and AfterField operation not chained?

### DISCUSSION THREAD

**Discussion by IoanAndrei97** [Laravel-Backpack/community-forum#300] [Q&A (Help)]
Hello guys,

There's any way to put afterColumn/afterField operations in array? Something like this:

`[
'name'      => 'brand',  
'type'      => 'text',
'label'     => 'Brand',
'afterColumn' => 'FooColumn',
]`

I want to do this because I'm trying to create methods that only return the columns definitions.

**Comment by karandatwani92** [comment]
You'll have to create your own custom helper to sort the Array. You can sort with `CRUD::addColumn($column_name)->afterColumn($after_column);` inside the loop after adding the column/fields.
